lib/npm.js

```diff
@@ -38,12 +40,12 @@ module.exports.install = (deps = [], opts = {}) => {
     cwd: opts.directory || process.cwd(),
     stdio: opts.silent ? 'ignore' : 'pipe'
   })
-}
+};
```

Comment:
Honestly I would move this package to `semistandard` if you feel like touching lint rules, but if not then I think you need to drop the semis for now.